lnbits/core/tasks.py

```diff
@@ -121,9 +121,18 @@ async def wait_for_paid_invoices(invoice_paid_queue: asyncio.Queue):
             async with httpx.AsyncClient(headers=headers) as client:
                 try:
                     r = await client.post(url, timeout=4)
-                    await mark_webhook_sent(payment, r.status_code)
-                except (httpx.ConnectError, httpx.RequestError):
-                    pass
+                    r.raise_for_status()
+                    await mark_webhook_sent(payment.payment_hash, r.status_code)
+                except httpx.HTTPStatusError as exc:
```

Comment:
Which is the difference between the two catch clauses (`httpx.HTTPStatusError` vs `httpx.RequestError`), why are they handled differently?

Reply:
why are they handled differently though?

Reply:
because we wannt to save the bad statuscode inside the payment details, it was like that before. so we now we did send it, but the receiver couldnt handle it.